src/main/java/co/rsk/federate/signing/hsm/message/ReleaseCreationInformation.java

```diff
@@ -1,16 +1,26 @@
 package co.rsk.federate.signing.hsm.message;
 
 import co.rsk.bitcoinj.core.BtcTransaction;
+import co.rsk.bitcoinj.core.Coin;
 import co.rsk.crypto.Keccak256;
+import co.rsk.peg.BridgeEvents;
+import co.rsk.peg.bitcoin.UtxoUtils;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
 import org.ethereum.core.Block;
+import org.ethereum.core.CallTransaction.Function;
 import org.ethereum.core.TransactionReceipt;
+import org.ethereum.vm.LogInfo;
 
 public class ReleaseCreationInformation {
     private final Block pegoutCreationBlock;
     private final TransactionReceipt transactionReceipt;
     private final Keccak256 pegoutCreationRskTxHash;
     private final BtcTransaction pegoutBtcTx;
     private final Keccak256 pegoutConfirmationRskTxHash;
+    private List<Coin> utxoOutpointValues = Collections.emptyList();
```

Comment:
This can be made `final` if adopted what I proposed before

Reply:
Awesome idea.

Reply:
Done